Buildings/Controls/OBC/Utilities/PIDWithAutotuning/SystemIdentification/FirstOrderTimedelayed/Validation/ControlProcessModel.mo

```diff
@@ -0,0 +1,70 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.SystemIdentification.FirstOrderTimedelayed.Validation;
+model ControlProcessModel "Test model for ControlProcessModel"
```

Comment:
Don't use instance name in comment

Reply:
Replaced all the instance names in comment.